ckanext/stadtzhharvest/harvester.py

```diff
@@ -285,6 +297,14 @@ def _import_package(self, harvest_object):
         package_dict['name'] = munge_title_to_name(package_dict[u'datasetID'])
         context = self._create_new_context()
 
+        # check if dataset must be deleted
+        import_action = package_dict.pop('import_action', 'update')
+        log.debug(import_action)
```

Comment:
Please consider removing the `log.debug` if they are "just" logging a variable without context. I am not sure how much use it has to someone who looks into the logs unless they are debugging.

Reply:
This change was not done by me. I removed it.